test/anvil.test.js

```diff
@@ -11,6 +11,8 @@ describe('1.8.9 anvil', () => {
     const inverse = Item.anvil(sword2, sword1, false, undefined)
     const finalItem = Item.fromNotch({ blockId: 276, itemCount: 1, itemDamage: 0, nbtData: { type: 'compound', name: '', value: { ench: { type: 'list', value: { type: 'compound', value: [{ lvl: { type: 'short', value: 1 }, id: { type: 'short', value: 34 } }] } }, RepairCost: { type: 'int', value: 3 } } } })
     const inverseFinalItem = Item.fromNotch({ blockId: 276, itemCount: 1, itemDamage: 0, nbtData: { type: 'compound', name: '', value: { ench: { type: 'list', value: { type: 'compound', value: [{ lvl: { type: 'short', value: 1 }, id: { type: 'short', value: 34 } }] } }, RepairCost: { type: 'int', value: 3 } } } })
+    finalItem.stackId = res.item.stackId
```

Comment:
Put these into the static constructor

Reply:
How would I assign the stack ID from the static functions? Or just set it to 0?

Reply:
See your API doc. Second parameter

Reply:
Ah, forgot that I did that. I should add it to the types too